web/profiles/joinup/src/Plugin/Block/AddContentBlock.php

```diff
@@ -73,9 +84,10 @@ public function build() {
     // caching ourselves.
     /** @var \Drupal\Core\Plugin\Context\Context[] $collection_contexts */
     $collection_contexts = $this->collectionContext->getRuntimeContexts(['collection']);
-    if ($collection_contexts['collection']->hasContextValue()) {
+    if ($collection_contexts && $collection_contexts['collection']->hasContextValue()) {
       $page_url = Url::fromRoute('custom_page.collection_custom_page.add', [
-        'rdf_entity' => $collection_contexts['collection']->getContextValue()->id(),
+        'rdf_entity' => $collection_contexts['collection']->getContextValue()
+          ->id(),
```

Comment:
This is not according to coding standards. Either the chained methods should be inline, or they should be on separate lines, not a combination of both.